pkg/boilerplate/operator/option.go

```diff
@@ -0,0 +1,28 @@
+package operator
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift/service-serving-cert-signer/pkg/boilerplate/controller"
+)
+
+// key is the singleton key shared by all events
+// the value is irrelevant, but pandas are awesome
+const key = "🐼"
```

Comment:
I like this, though I wonder if it ever would show up in a log & confuse someone.

Reply:
Sounds like the greatest log message ever :panda_face: